cluster-autoscaler/cloudprovider/openshiftmachineapi/machineapi_controller.go

```diff
@@ -38,6 +39,7 @@ const (
 	machineProviderIDIndex = "openshiftmachineapi-machineProviderIDIndex"
 	nodeProviderIDIndex    = "openshiftmachineapi-nodeProviderIDIndex"
 	defaultMachineAPI      = "v1beta1.machine.openshift.io"
```

Comment:
nit: can we consolidate
`defaultMachineAPI`, `apiVersionGroupEnvVar` and `MACHINEAPI_RESOURCE_VERSION_GROUP` as `defaultMachineAPIVersionGroup`, `apiVersionGroupEnvVar`, `MACHINEAPI_VERSION_GROUP`